pkg/apis/cyclone/v1alpha1/project.go

```diff
@@ -0,0 +1,47 @@
+package v1alpha1
+
+import (
+	core_v1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Project defines a project which holds a lot of common default information of some workflow under it.
+type Project struct {
+	// Metadata for the resource, like kind and apiversion
+	meta_v1.TypeMeta `json:",inline"`
+	// Metadata for the particular object, including name, namespace, labels, etc
+	meta_v1.ObjectMeta `json:"metadata,omitempty"`
+	// Spec is the Workflow specification
+	Spec ProjectSpec `json:"spec"`
+}
+
+// ProjectSpec defines project specification.
+type ProjectSpec struct {
+	// Services contains default value of various type of integrations.
+	Services []ServiceItem `json:"services"`
```

Comment:
`service` is not very good as it is easy confused with k8s service and dependent services in integration test.

Reply:
Indeed, any suggestions ?